xapian-letor/ranker/ranker.cc

```diff
@@ -106,9 +106,13 @@ load_list_fvecs(const string & filename)
 	train_file.ignore();
 
 	fv.set_score(0);
-	fvv.push_back(fv);
+	fvv[qid].push_back(fv);
     }
-    return fvv;
+    std::vector<vector<FeatureVector>> vt;
+    for (auto it = fvv.begin(); it != fvv.end(); ++it) {
```

Comment:
Another place that where C++11 range `for` would be more readable.

Reply:
fixed in a fixup commit.